src/gordon_gcp/plugins/service/event_consumer.py

```diff
@@ -365,7 +376,10 @@ def _get_and_validate_pubsub_msg_schema(self, pubsub_msg_data):
             pubsub_msg, event_msg_data, schema)
 
         msg_logger.debug(f'Adding message to the success channel.')
-        await self.success_channel.put(event_msg)
+        event_msg.update_phase(self.start_phase)
```

Comment:
Double-pasting this comment since it got hidden when I commented commit by commit.

This is my bad. I actually meant to pass start_phase on line 327/338 [(if the link works](https://github.com/spotify/gordon-gcp/pull/37/files#diff-ba365bd6efd92a959016bfbec05ccc0fR338)) so that it's initialized with the phase, but it looks like I only added it to the log message instead. I think it makes more sense to initialize it with a start phase rather than update it with one.

Reply:
Ah I like that better - I'll change it

Reply:
@descent3me is this what you had in mind? (updated the commit message too)